packages/cells/src/widget.ts

```diff
@@ -1098,6 +1101,70 @@ export class CodeCell extends Cell<ICodeCellModel> {
     model.stateChanged.connect(this.onStateChanged, this);
   }
 
+  /**
+   * Observe the movement of the caret in the output area.
+   *
+   * Emits scroll request if the caret moved.
+   */
+  private _observeCaretMovementInOuput(output: OutputArea) {
+    let lastOnCaretMovedHandler: () => void;
+    let lastTarget: HTMLElement | null;
+
+    output.node.addEventListener('keydown', (e: KeyboardEvent) => {
```

Comment:
Should this listener be removed when disposing the cell?

Reply:
Done in e458d5586af80179995a3e4e4e386317f52e4846.